manifest.json

```diff
@@ -1,12 +1,20 @@
 {
   "name": "Evolv AI",
   "description": "Display information about Evolv AI optimizations",
-  "version": "1.0",
+  "version": "1.2",
   "manifest_version": 3,
   "host_permissions": [
-
+    "https://participants.evolv.ai/*",
+    "https://participants-stg.evolv.ai/*"
   ],
-  "permissions": [],
+  "permissions": [
```

Comment:
test removing these

Reply:
Looks like it doesn't need these anymore. They were required for a different implementation I tried.